primitives/src/policy.rs

```diff
@@ -180,8 +186,17 @@ impl Policy {
             .state_chunks_max_size
     }
 
+    #[inline]
+    pub fn get_or_init(policy: Policy) -> Policy {
+        *GLOBAL_POLICY.get_or_init(|| policy)
+    }
+}
+
+#[cfg_attr(feature = "ts-types", wasm_bindgen::prelude::wasm_bindgen)]
```

Comment:
Instead of having the long fully qualified import path everywhere, you can conditionally import a package only when a feature is active. That way you can use it with it's short direct name.